### PR TITLE
Move CustomFasSyncInterval to Beta

### LINKED ISSUE #2646: Move CustomFasSyncInterval to Beta
**Is your feature request related to a problem? Please describe.**
The "Custom resync period for FleetAutoscaler" feature for the Fleet Autoscaler has been in Alpha since 1.17.0.

It did have some initial stability issues, but have [since been rectified](https://github.com/googleforgames/agones/pull/2422) and has been stable in e2e tests since then.

**Describe the solution you'd like**

Seems like an appropriate time to move to Beta to get some more testing on the feature before moving it to Stable.

**Describe alternatives you've considered**

Remove the functionality, but it seems useful, so I suggest keeping it.

**Additional context**

Original issue: https://github.com/googleforgames/agones/issues/1955
Reference docs: 
* https://agones.dev/site/docs/reference/fleetautoscaler/
* https://agones.dev/site/docs/reference/agones_crd_api_reference/#autoscaling.agones.dev/v1.FleetAutoscaler

### PR DESCRIPTION
**Pull request by govargo**
**What type of PR is this?**

/kind feature

**What this PR does / Why we need it**:

This PR bump up CustomFasSyncInterval from Alpha to Beta.

**Which issue(s) this PR fixes**:

Closes #2646

**Special notes for your reviewer**:

None

### CODE REVIEW

**Comment by govargo** [comment]
Pending: https://github.com/googleforgames/agones/issues/2646#issuecomment-1173600375

**Comment by markmandel** [comment]
@govargo did you want review on this? Just figured I would ask first since it had "WIP" in the title 😄 

Lemme know if this PR is ready for review or not.

**Comment by govargo** [comment]
@markmandel Thank you for your comment.
I removed WIP from this PR title and rebased latest main repository.
Now, it's ready for review.

File: site/content/en/docs/Guides/feature-stages.md [hunk 1]
```diff
@@ -29,7 +29,7 @@ The current set of `alpha` and `beta` feature gates are:
 |--------------|---------|---------|-------|-------|
 | Example Gate (not in use) | `Example` | Disabled | None | 0.13.0 |
 | [Player Tracking]({{< ref "/docs/Guides/player-tracking.md" >}}) | `PlayerTracking` | Disabled | `Alpha` | 1.6.0 |
-| [Custom resync period for FleetAutoscaler](https://github.com/googleforgames/agones/issues/1955) | `CustomFasSyncInterval` | Disabled | `Alpha` | 1.17.0 |
+| [Custom resync period for FleetAutoscaler](https://github.com/googleforgames/agones/issues/1955) | `CustomFasSyncInterval` | Enabled | `Beta` | 1.25.0 |
```
Comment (markmandel) [inline site/content/en/docs/Guides/feature-stages.md:32]: You'll see to do a feature shortcode the table:

https://agones.dev/site/docs/contribute/#within-a-page

File: site/content/en/docs/Reference/fleetautoscaler.md [hunk 2]
```diff
@@ -37,7 +37,7 @@ spec:
       # maximum fleet size that can be set by this FleetAutoscaler
       # required
       maxReplicas: 20
-  # [Stage:Alpha]
+  # [Stage:Beta]
```
Comment (markmandel) [inline site/content/en/docs/Reference/fleetautoscaler.md:40]: You will also need to feature shortcode the example:

https://agones.dev/site/docs/contribute/#within-a-page

**Review by markmandel — COMMENTED** [review]
Just doing a quick search, also found:
 https://github.com/googleforgames/agones/blob/73cdabc4bf0f238ce2d913df8352d11b3a888445/examples/fleetautoscaler.yaml#L50-L53

We will need to regen the crd reference (the tests will fail) with the yaml update:
https://github.com/googleforgames/agones/blob/main/build/README.md#make-gen-api-docs

https://github.com/googleforgames/agones/blob/73cdabc4bf0f238ce2d913df8352d11b3a888445/pkg/apis/autoscaling/v1/fleetautoscaler.go#L57-L62

File: site/content/en/docs/Guides/feature-stages.md (hunk 1 above)
Comment (govargo) [reply site/content/en/docs/Guides/feature-stages.md:43]: Added feature shortcode. If more fixs are needed, please let me know.

**Comment by govargo** [comment]
Done `make gen-api-docs`

**Review by markmandel — APPROVED** [review]
LGTM!

**Pull request merged**